conf/messages.en

```diff
@@ -482,7 +482,7 @@ liability.expectToExceedThreshold.two.required.fields = Date must include the {0
 liability.expectToExceedThreshold.one.field = Date must include the {0}
 
 liability.exceededThresholdWeightDate.sectionHeader = Liability details
-liability.exceededThresholdWeightDate.title = When did you you meet the 10,000kg threshold?
+liability.exceededThresholdWeightDate.title = When did you meet the 10,000kg threshold?
```

Comment:
is this correct in welsh?

Reply:
Looks like it as there aren't any repeated words... But I'm clearly not Welsh.

Pryd gwnaethoch gyrraedd y trothwy 10,000kg?

Reply:
<img width="688" alt="Screenshot 2022-07-14 at 13 46 03" src="https://user-images.githubusercontent.com/35451824/178985572-007f626d-935e-40c2-b479-c295705e7147.png">
Aye looks good